script/docker-start

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e
+
+source $(dirname $0)/generatecert
+
+exec redoctober -addr=:8080 $@
```

Comment:
This is the port within the Docker container—the user of the image can map it to whatever port externally that they like. Does that work?